aiocache/decorators.py

```diff
@@ -111,6 +117,13 @@ async def decorator(
 
         result = await f(*args, **kwargs)
 
+        if self.value_checker:
+            if not isinstance(self.value_checker(result), bool):
+                raise TypeError('value_checker callable should return bool')
```

Comment:
I'd be inclined to leave this to type checkers (e.g. mypy) and not care about runtime checks here.

Reply:
Done